client-src/elements/chromedash-typeahead.ts

```diff
@@ -363,12 +378,16 @@ export class ChromedashTypeaheadDropdown extends SlDropdown {
         if (event.key === 'ArrowDown' && menuItems[currentItemIndex + 1]) {
           this.setCurrentItem(menuItems[currentItemIndex + 1]);
         }
-        if (event.key === 'ArrowUp' && menuItems[currentItemIndex - 1]) {
+        if (
+          event.key === 'ArrowUp' &&
```

Comment:
Instead of doing this additional check with currentItemIndex here and the change to the index below, you could filter out the invisible item before. Then the code here could stay as-is.

Reply:
I feel the current comparison seems simpler than filtering.